RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java

```diff
@@ -40,153 +40,181 @@ public StoreController(StoreService storeService) {
-        storeService.addProductToStore(store, product);
-        LOGGER.info(String.format("Product %d was added to Store %d", product.getId(), store.getId()));
+    public void addProductToStore(@PathVariable Integer storeId, @PathVariable Integer productId)
+            throws DataNotFoundException {
+        storeService.addProductToStore(storeId, productId);
+        LOGGER.info(String.format("Product %d was added to Store %d", productId, storeId));
     }
 
-    @PostMapping("/manyProducts/")
-    @ResponseStatus(value = HttpStatus.CREATED)
-    public void addProducstToStore(@RequestBody Store store, Product product) {
-        storeService.addProductToStore(store, product);
-        LOGGER.info(String.format("Product %d was added to Store %d", product.getId(), store.getId()));
+    /**
+     * This method adds products to store
+     *
+     * @param productsId list of productId that will be added to store
+     * @param storeId    store where products will be added
+     * @throws DataNotFoundException if list of productId is empty, or store is not found
+     */
+    @PostMapping("/manyProducts/{userId}/{storeId}")
+    @ResponseStatus(value = HttpStatus.OK)
+    public void addProductsToStore(@RequestBody List<Integer> productsId, @PathVariable Integer storeId,
+                                   @PathVariable Integer userId) throws DataNotFoundException {
+
+        storeService.addProductsToStore(productsId, storeId);
+        LOGGER.info(String.format("Products  where added to Store %d", storeId));
     }
 
+
+    /**
+     * This method retrieves products that are not presented in the store. This
+     * method is used for adding products to store
+     *
+     * @param storeId - store where we look for products, that are not added there yet
+     * @param userId  - user whose products and stores we are looking for
+     * @return - set of products that are not added to store
+     * @throws DataNotFoundException - if result set is empty
+     */
     @GetMapping("/{storeId}/notMappedProducts/{userId}")
     @ResponseBody
     @ResponseStatus(value = HttpStatus.OK)
     public Set<Product> getNotMappedProducts(@PathVariable Integer storeId, @PathVariable Integer userId,
-                                             HttpServletResponse response) {
-        try {
-            Set<Product> notMappedProducts = storeService.getNotMappedProducts(storeId, userId);
-
-            LOGGER.info("All Products were found");
-            return notMappedProducts;
-        } catch (DataNotFoundException e) {
-            response.setStatus(HttpServletResponse.SC_NOT_FOUND);
-            LOGGER.error(e.getMessage());
-            return null;
-        }
+                                             HttpServletResponse response) throws DataNotFoundException {
+
+        Set<Product> notMappedProducts = storeService.getNotMappedProducts(storeId, userId);
+
+        LOGGER.info("All Products were found");
```

Comment:
Yep, I understand, I propose another the message text, because your message is useless